crates/pop-parachains/src/call/metadata/action.rs

```diff
@@ -43,19 +43,19 @@ pub enum Action {
 		props(Pallet = "Assets")
 	)]
 	MintAsset,
-	/// Create an NFT collection.
+	/// Create a NFT collection.
```

Comment:
I guess 'a NFT'  is grammatically correct, but I’ve seen 'an NFT' being used frequently. For example: [Polkadot Wiki](https://wiki.polkadot.network/docs/learn-nft-projects#rmrk).